Update file1.adoc (#16)
alxlenc/pygh-tests: content/file1.adoc

## This is a Title
I create a file.

> A link to the list of drivers and their snapshot support is provided in the references section.

Reviewer: How about...
```suggestion
In the references section, you'll find a link leading to a list of drivers accompanied by their snapshot support.
```
